### PR TITLE
Added a tutorial to illustrate how to copy local files to azure blob …

### PR DESCRIPTION
**Pull request by jasonweiyi**
…and process the uploaded files in MBrace cluster.

### CODE REVIEW

**Comment by krontogiannis** [comment]
Hi!

A few observations:
* I believe it's not necessary to use the Azure SDK APIs to create a container first (lines 49 to 52). You can use the CloudDirectory API. (Also the Azure implementation implicitly creates the container if it doesn't exist).
* The is an alternative to using RunLocally for store operations. It's the cluster.StoreClient.* APIs. Maybe you could add this too.

File: azure/HandsOnTutorial/11-upload-file-to-blob.fsx [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+*)
+
+(** First you connect to the cluster: *)
+let cluster = Runtime.GetHandle(config)
+
+cluster.ShowProcesses()
+cluster.ShowWorkers()
+
+
+(** Create a tmp file locally, wrote some random content, and return the file path. *)
```
Comment (dsyme) [inline azure/HandsOnTutorial/11-upload-file-to-blob.fsx]: Please write dialog in the form "Now you ...", "Next you ..." etc. and use full English sentences like a book.

"The following function creates a temporary file locally, containing some random content, and returns the file path."

"Next you create a local temporary file."

"Next you use the Azure SDK to create a temporary container  to contain blobs."

etc.

File: azure/HandsOnTutorial/11-upload-file-to-blob.fsx [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+DeleteTempFile tmpFile
+
+(** Cound the lines of the uploaded file in the MBrace cluster. *)
+let lines = 
+    CloudFile.ReadAllLines cFile.Path 
+    |> cluster.Run
+    |> Array.length
+
+
+
```
Comment (dsyme) [inline azure/HandsOnTutorial/11-upload-file-to-blob.fsx]: Always finish each sample with a short summary of what has been learned.  See the other samples for the phrasing to use (use a similar style of phrasing for each one)

File: azure/HandsOnTutorial/11-upload-file-to-blob.fsx [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+    |> cluster.RunLocally
+
+(** Delete the local temp file *)
+DeleteTempFile tmpFile
+
+(** Cound the lines of the uploaded file in the MBrace cluster. *)
+let lines = 
+    CloudFile.ReadAllLines cFile.Path 
+    |> cluster.Run
+    |> Array.length
```
Comment (dsyme) [inline azure/HandsOnTutorial/11-upload-file-to-blob.fsx]: This Array.length is running on the client, I think you want 

    cloud { 
        let! lines = CloudFile.ReadAllLines cFile.Path 
        return lines.Length 
    } |> cluster.Run

**Comment by jasonweiyi** [comment]
I have integrated your comments.

File: azure/HandsOnTutorial/11-upload-file-to-blob.fsx [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+(** Delete a file given its path *)
+let DeleteTempFile path =
+    File.Delete path
+
+(** Create a local temp file. *)
+let tmpFile = CreateTempFile()
+
+
+(** Next, you upload the created file to the tmp container in Azure blob storage. The tmp container
+will be created if it does not exist. Note the use of the local {...} expression and the cluster.RunLocally method:
```
Comment (dsyme) [inline azure/HandsOnTutorial/11-upload-file-to-blob.fsx]: You can used markdown formatting in the comments, e.g. 

    ``local { .. }``

**Comment by dsyme** [comment]
This looks good to me!

**Comment by dsyme** [comment]
Oh - one last thing - please add the file to the relevant project/solution file as well so people can find it :)

**Pull request merged**